ansible/roles/installation/steamvr/tasks/main.yml

```diff
@@ -0,0 +1,90 @@
+---
+- name: Update package lists
+  apt: update_cache=yes
+  become: yes
+  
+- name: Intall needed libs for steamvr
+  apt: name={{ item }} install_recommends=no
+  become: yes
+  with_items:
+    - curl
+    - ca-certificates
+    - lib32gcc1
+    - binutils
+    - xdg-utils
+    - software-properties-common
+    - sudo
+    - libvulkan1
+    - usbutils
+    - libcap2-bin
+
+- name: Update and upgrade apt packages
+  become: yes
+  apt:
+    upgrade: dist
+    update_cache: yes
+
+- name: Gather package facts
+  package_facts:
+    manager: apt
+
+- name: Install libudev0_175 from the internet if absent
+  apt:
+    deb: http://mirrors.kernel.org/ubuntu/pool/main/u/udev/libudev0_175-0ubuntu9_amd64.deb
+    state: present
+  become: yes
+  when: '"libudev0" not in ansible_facts.packages'
+
+- name: Create steamcmd directory
+  file:
+    path: "/home/{{ user }}/steamcmd"
```

Comment:
introduce user_home_dir varible in defaults and use it instead of composing path.

Reply:
done